llvm/tools/llvm-pdbutil/llvm-pdbutil.cpp

```diff
@@ -865,6 +869,19 @@ static void yamlToPdb(StringRef Path) {
     }
   }
 
+  std::vector<object::coff_section> Sections;
+  if (!Dbi.SectionHeaders.empty()) {
+    for (const auto &Hdr : Dbi.SectionHeaders) {
+      Sections.emplace_back(Hdr.toCoffSection());
```

Comment:
One liners need no braces for `for`, see https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements